docs/releaseNotes.yml

```diff
@@ -35,7 +35,12 @@ items:
     prevVersion: 2.1.0
     date: 'TBD'
     notes:
-      # next changelog item here
+      - title: Docker BuildKit always used for builds
+        type: change
+        body: >-
+          Docker BuildKit is enabled for all Emissary builds. Additionally, the Go
+          build cache is fully enabled when building images, speeding up repeated builds.
+        docs: https://docs.docker.com/develop/develop-images/build_enhancements/
```

Comment:
The buildkit syntax extensions reference (https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/syntax.md) is probably a better link than Docker's how-to guide. Especially since Docker's guide refers to Docker's Dockerfile syntax reference, which _doesn't_ mention the buildkit extensions.

Reply:
👍